src/mono/wasm/debugger/DebuggerTestSuite/DebuggerTestBase.cs

```diff
@@ -832,15 +835,15 @@ internal async Task<JToken> GetObjectOnFrame(JToken frame, string name)
         }
 
         // Find an object with @name, *fetch* the object, and check against @o
-        internal async Task<JToken> CompareObjectPropertiesFor(JToken locals, string name, object o, string label = null, int num_fields = -1)
+        internal async Task<JToken> CompareObjectPropertiesFor(JToken locals, string name, object o, string label = null, int num_fields = -1, bool skip_num_fields_check = false)
         {
             if (label == null)
                 label = name;
             var props = await GetObjectOnLocals(locals, name);
             try
             {
                 if (o != null)
-                    await CheckProps(props, o, label, num_fields);
+                    await CheckProps(props, o, label, num_fields, skip_num_fields_check);
```

Comment:
you could reflect in the test ;)

Reply:
I tried that, and it's a little messy. And the number of members being returned will also change soon (to include private ones), so I'll have to stick with this check!